pom.xml

```diff
@@ -56,6 +56,12 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
```

Comment:
But then I cant use the @Slf4j annotation, because its from lombok dependency. In my pov for logging purposes its harmless.

Reply:
And its rly annoying to write every time that boiler plate static LOGGER attribute.

Reply:
I don't get it. I assumed that we don't use lombok on models bc its dangerous, but for logging why can't we use it?

Reply:
As mentioned previously on slack, we want to discourage the use of lombok, as we want the focus to be on learning to write good code. This involves both the actual code part (for lack of a better term) and on the configs that are used.
For example here, there are multiple things that lombok will do in the background:
- add all the configs needed in order for the logging framework to work as expected
- choose an implementation
- create the logger entity in the class (I have not used field/constant on purpose in order to not reveal how it would be defined otherwise)

As such, there are a few steps that lombok will provide out of the box, that were skipped otherwise.
Therefore, I would like to encourage setting up the logging part without lombok, in order to see how that would be done.

Reply:
Fixed. New code available.